testbin/setup-envtest.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
```

Comment:
why did we add this file?

Reply:
it shouldn't be there. Removed `testbin` directory.